include/materials/material_utility.tcc

```diff
@@ -0,0 +1,230 @@
+//! Compute mean stress p
+inline const double mpm::material_utility::compute_mean_p(
+    const Eigen::Matrix<double, 6, 1>& stress) {
+  // Compute mean p in compresion positive
```

Comment:
Could we move this comment to the top, as this isn't adding any additional value here.